### ISSUE TITLE
Is there some function to generate a vector representation for an image using one of the trained models?

### ISSUE THREAD

**Issue by sogituser** [jwyang/JULE-Torch#8]
I'd like to use this feature representation for further clustering. 

Thanks in advance!

**Comment by jwyang** [comment]
Hi, I have all the features saved. You can download them from:

https://drive.google.com/open?id=0B9J-9A2jotGRMkJ0SExaWnNYUTA

**Comment by albertyou2** [comment]
@jwyang @sogituser 
hi
i think the question is : how to extract features from a image by using JULE-Torch and then save them to a vector(numpy ,etc). i also concern this  job too.is there any api to do this ?
thank you very much !

**Comment by sogituser** [comment]
Hey, Thanks for a quick response! But I want to extract features of images that I have.. That is how can I use your trained model to extract image(my dataset) features.

**Comment by jokereactive** [comment]
Yes, same question. Is there available code to extract features <vectors> from my own set of images using a model trained on one of your cited datasets?

**Comment by jwyang** [comment]
I did not add that code to the repository, but it should be very easy to do that.

**Comment by jeannie14** [comment]
@jwyang can you please add the code to use model trained using approach to extract features? That will help to try the approach out on datasets of other domains. Thanks!
